zaas-client/src/main/java/org/zowe/apiml/zaasclient/service/internal/ZaasHttpsClientProvider.java

```diff
@@ -129,13 +146,21 @@ private KeyStore getEmptyKeystore() throws KeyStoreException, CertificateExcepti
     }
 
     private InputStream getCorrectInputStream(String uri) throws IOException {
-        if (uri.startsWith(SAFKEYRING + "://")) {
-            URL url = new URL(replaceFourSlashes(uri));
+        if (isKeyring(uri)) {
+            URL url = new URL(formatKeyringUrl(uri));
             return url.openStream();
         }
         return new FileInputStream(uri);
     }
 
+    /**
+     * Replaces 4 slashes on 2 in URI
+     *
+     * @param storeUri - URI as {@link String}
+     * @return same URI, but with 2 slashes, or null, if {@code storeUri} is null
+     * @deprecated replaced by {@link #formatKeyringUrl}
+     */
+    @Deprecated
     public static String replaceFourSlashes(String storeUri) {
```

Comment:
I guess all the `replaceFourSlashes`  occurrences could be directly removed, as I see it's used only once in a test.

Reply:
Up to you - removed